src/crypto/tls/handshake_server_tls13.go

```diff
@@ -248,6 +293,9 @@ GroupSelection:
 	}
 
 	c.serverName = hs.clientHello.serverName
+
+	hs.handshakeTimings.ReadClientHello = hs.handshakeTimings.elapsedTime()
```

Comment:
This is too late, it also includes time spend on generating a key share. Maybe ReadClientHello should be called `ProcessClientHello` (or be documented to be more than just receiving the CH)?

Reply:
I think we want to this in its current location, though of course that may change. Fixed by renaming for now!